### PR TITLE
Update website config to current branches

### PR DESCRIPTION
**Pull request by SenseException**

### CODE REVIEW

File: .doctrine-project.json [hunk 1]
```diff
@@ -10,11 +10,23 @@
             "slug": "latest",
             "upcoming": true
         },
+        {
+            "name": "0.3",
+            "branchName": "0.3.x",
+            "slug": "0.3",
+            "upcoming": true
```
Comment (SenseException) [inline .doctrine-project.json]: There's already a 0.3 tag, so I don't know if it's an upcoming or a current version. The default branch is still 0.2.x.

Comment (greg0ire) [reply .doctrine-project.json]: It's current. The default branch should be changed to 0.3.x, and I lacked the necessary permissions for that. Can you please do it?

Comment (SenseException) [reply .doctrine-project.json]: Looks like I haven't the permissions either.

**Comment by SenseException** [comment]
I've set 0.3.x as the current version in the config.

**Review by greg0ire — APPROVED** [review]

**Pull request merged**